src/components/Question/Question.tsx

```diff
@@ -36,17 +36,19 @@ export const Question: React.FC = () => {
         </div>
 
         <div className="buttons-box">
-          <button className='back-btn btn'>back</button>
-          <button className='next-btn btn'>next</button>
-            {/*onClick={()=>updateAllAnswers({...answerContext, [currentQuestion]: answerInput.answer})}*/}
+          {currentAnswer && <button className='back-btn btn'>back</button>}
```

Comment:
I created ticket #48  to fix the styling issue

Reply:
Is that issue where the next button goes where the back button is?